### DISCUSSION TITLE
ib_insync -> ib_async

### DISCUSSION THREAD

**Discussion by juggersworth** [ib-api-reloaded/ib_async#59] [General]
I've been developing/using a bot written with ib_insync for a couple years. I was sad when I read of Ewald's passing. He was so patient and helpful to everyone! Wondering if moving over is a straight shot, ie: change 'from ib_insync import...' to 'from ib_async import...' and away we go, or have changes been made that would reflect in the code? I'll create a fork and try it at some point in the near future, but thought I'd ask. Thanks for your efforts to maintain this code base. It's really helpful to many people.

**Comment by mattsta** [comment]
> Wondering if moving over is a straight shot, ie: change 'from ib_insync import...' to 'from ib_async import...' 

Yes, only the name has changed so far. All of the objects and imports and interfaces remain the same. We had to rename the project because, sadly, nobody has access to the original pypi packages anymore.

Any interface breaking changes would be under a new major version number (2.x), but I think we can continue to add new features and behavior changes as optional configuration parameters (or extended function arguments defaulting to legacy values) instead of disturbing people's current workflows.

Thanks for reaching out. Feel free to ask about anything else if your own usage could be made simpler through better APIs or results or fixing edge cases in things.
